packages/epo-react-lib/.storybook/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "commonjs"
```

Comment:
need this stubbed out package.json otherwise Storybook tries to run using the parent package.json which uses `"type": "module"` and requires using `.cjs` filetypes